pandas/io/formats/style.py

```diff
@@ -65,6 +65,8 @@ class Styler(object):
         a unique identifier to avoid CSS collisions; generated automatically
     caption: str, default None
         caption to attach to the table
+    all_ids: bool, default False
+        if True, each cell will have an ``id`` attribute in their HTML tag.
```

Comment:
Capitalize `if`.

Could you include the form these take? I believe it's `T_<uuid>_row<i>_col<j>` for values in the table.